pom.xml

```diff
@@ -44,6 +44,10 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-security</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework</groupId>
+			<artifactId>spring-web</artifactId>
+		</dependency>
```

Comment:
If we are using this for both test & compile scopes do we need to define it twice. My thinking is you can have one and not specify the scope.

Reply:
Makes sense. Thanks, let me remove the dependency for the test.